dnf.spec

```diff
@@ -174,6 +174,9 @@ mv $RPM_BUILD_ROOT%{_bindir}/dnf-automatic-2 $RPM_BUILD_ROOT%{_bindir}/dnf-autom
 rm $RPM_BUILD_ROOT%{_bindir}/dnf-automatic-3
 %endif
 
+# This will eventually be the new default location for repo files
+mkdir $RPM_BUILD_ROOT%{_sysconfdir}/distro.repos.d
```

Comment:
you can have `mkdir $RPM_BUILD_ROOT%{_sysconfdir}/distro.repos.d` in spec downstream in Mageia. The purpose of having %ghost is to not create that directory so for F24 it's still the `/etc/yum.repos.d/` dir.